packages/addons/service/tvheadend42/source/bin/tv_grab_file

```diff
@@ -37,7 +37,14 @@ then
       exec "$XMLTV_LOCATION_SCRIPT"
     fi
   elif [ "$XMLTV_TYPE" = "WEB" ]; then
-    wget -qO - "$XMLTV_LOCATION_WEB"
+    case "$XMLTV_LOCATION_WEB" in
+    *.gz | *.bz2 | *.xz)
```

Comment:
did you tested it with xz and bz2 ? wondering if gunzip is actually able to do that at all

Reply:
you are right. I did this in the morning before I started my real-life work. I hope I am not such a bad developer, when I am at work :|
piping the wget output into zcat should fix this and handle all types of data - equally to the handler for compressed files. This bash script is not DRY know - to avoid other sideeffects I did not change any other parts of the script and it is still short and easy readable. Sorry for annoying you with that minor change, @chewitt

Reply:
we use busybox zcat, that looks like it supports bz2 and xz
`wget -qO - https://tukaani.org/xz/xz-5.2.5.tar.xz | zcat`
`wget -qO - https://tukaani.org/xz/xz-5.2.5.tar.bz2 | zcat`

looks working :)

Reply:
@mhutzsch it looks working (I have nothing for testing in real world), its okay to merge ?

Reply:
Yeah, merging should have no negative side effect.